Sources/UIView.swift

```diff
@@ -312,6 +312,10 @@ open class UIView: UIResponder, CALayerDelegate, UIAccessibilityIdentification {
     open func addGestureRecognizer(_ recognizer: UIGestureRecognizer) {
         recognizer.view = self
         gestureRecognizers.insert(recognizer)
+
+        if gestureRecognizers.isEmpty {
```

Comment:
how could this ever happen? wouldn't gestureRecognizers always contain at least one item at this point?

Reply:
good catch, the logic doesn't make much sense. This is what was intended https://github.com/flowkey/UIKit-cross-platform/pull/333/commits/4256b9f28704baeb2278b12e1558d6486841a21b